st3/sublime_lib/_compat/typing.py

```diff
@@ -1,4 +1,6 @@
-try:
+import sys
+
+if sys.version_info >= (3, 8, 0):
     from typing import *  # noqa: F401, F403
-except ImportError:
+else:
     from .typing_stubs import *  # type: ignore # noqa: F401, F403
```

Comment:
Should it be something like this instead so that the Sublime Text `typing` depedency is used when installed?

```py
if sys.version_info >= (3, 8, 0):
    from typing import *  # noqa: F401, F403
else:
    try:
        from typing import *  # noqa: F401, F403
    except ImportError:
        from .typing_stubs import *  # type: ignore # noqa: F401, F403
```

Although I'm not sure what would be the point really. It would not change the runtime behavior and is not needed for pyright either.

Reply:
Probably not needed, and we've had issues with a similar approach with `enum`.